checklists/clgmap.php

```diff
@@ -15,78 +15,105 @@
+                  closeAllInfoWins(); 
+                  infoWin.open(map.mapLayer, m); 
+               });
+
+               google.maps.event.addListener(m, "dblclick", function(){ 
+                 closeAllInfoWins();
+                 m.setAnimation(google.maps.Animation.BOUNCE);
+                 navigateToCheckList(checklistId, pid);
+               });
+            }
+
+            map.mapLayer.fitBounds(bounds);
+         }
 
 			function initialize(){
-				var dmLatLng = new google.maps.LatLng(41.0, -95.0);
-				var dmOptions = {
-					zoom: 13,
-					center: dmLatLng,
-					mapTypeId: google.maps.MapTypeId.TERRAIN
-				};
-
-				map = new google.maps.Map(document.getElementById("map_canvas"), dmOptions);
-				<?php
-				$clArr = $clManager->getResearchPoints();
-				foreach($clArr as $clid => $inArr){
-					echo "var point".$clid." = new google.maps.LatLng(".$inArr['lat'].", ".$inArr['lng'].");\n";
-					echo "points.push( point".$clid." );\n";
-					echo 'var marker'.$clid.' = new google.maps.Marker({ position: point'.$clid.', map: map, title: "'.$inArr['name'].'" });'."\n";
-					//Single click event
-					echo 'var infoWin'.$clid.' = new google.maps.InfoWindow({ content: "<div style=\'width:300px;\'><b>'.$inArr['name'].'</b><br/>Double Click to open</div>" });'."\n";
-					echo "infoWins.push( infoWin".$clid." );\n";
-					echo "google.maps.event.addListener(marker".$clid.", 'click', function(){ closeAllInfoWins(); infoWin".$clid.".open(map,marker".$clid."); });\n";
-					//Double click event
-					if($target == 'keys'){
-						echo "var lStr".$clid." = '../ident/key.php?clid=".$clid."&pid=" . $pid."&taxon=All+Species';\n";
-					}
-					else{
-						echo "var lStr".$clid." = 'checklist.php?clid=".$clid."&pid=" . $pid."';\n";
-					}
-					echo "google.maps.event.addListener(marker".$clid.", 'dblclick', function(){ closeAllInfoWins(); marker".$clid.".setAnimation(google.maps.Animation.BOUNCE); window.location.href = lStr" . htmlspecialchars($clid, HTML_SPECIAL_CHARS_FLAGS) . "; });\n";
-				}
-				?>
-				resizeMap();
-			}
 
-			function resizeMap() {
-				var minLng = 180;
-				var minLat = 180;
-				var maxLng = -180;
-				var maxLat = -180;
-
-				// Find the max/min points
-				for( var i = 0; i < points.length; i++ ) {
-					var p = points[i];
-					if ( p.lat() < minLat ) minLat = p.lat();
-					if ( p.lat() > maxLat ) maxLat = p.lat();
-					if ( p.lng() < minLng ) minLng = p.lng();
-					if ( p.lng() > maxLng ) maxLng = p.lng();
-				}
-				var swLatLng = new google.maps.LatLng(minLat, minLng);
-				var neLatLng = new google.maps.LatLng(maxLat, maxLng);
-				if(minLat == maxLat && minLng == maxLng){
-					map.setCenter(swLatLng);
-				}
-				else{
-					var llBounds = new google.maps.LatLngBounds(swLatLng, neLatLng);
-					map.fitBounds(llBounds);
-				}
-			}
+            //Load Server Data from HTML Data Attributes
+            $('.service-container').each(function() {
```

Comment:
I noticed this in your other PR as well. I think that there's a general desire to move away from jQuery, but if there is no easy workaround, I personally have no issue this with.

Reply:
Okay, I will see if I can use something else here like document selection

Reply:
Okay no more jquery :)